tensortrade/__init__.py

```diff
@@ -12,5 +12,6 @@
 from . import env
 from . import stochastic
 from . import agents
+from . import binance
```

Comment:
You cannot just load this in here.

Reply:
Indeed. Delete this file. It is self-contained

Reply:
How should I do it?

Reply:
> You cannot just load this in here.

Fixed